### ISSUE TITLE
PB position issue when docstring has an empty line at the end

### ISSUE THREAD

**Issue by rlisahuang** [UCSD-PL/vscode#27]
To replicate: 
```py
import itertools
import matplotlib.pyplot as plt

"""
one line

"""

data = [180,250,130,50]

fig, ax = plt.subplots()

# plot the data using a bar chart with different colors on each bar
## ---
ax.bar(range(len(data)), data, color=['#FF0000','#00FF00','#0000FF','#FFFF00'])

# add a title and a label for the y axis
ax.set_title('Data')
ax.set_ylabel('Value')

# add a label for each bar
ax.set_xticks(range(len(data)))
ax.set_xticklabels(['A','B','C','D'])

# add a grid
ax.grid(True)

# show the plot
## ---
```

**Comment by rlisahuang** [comment]
More bugs exposed by docstrings (even without an empty line at the end). Need to look into this more.

**Comment by rlisahuang** [comment]
Two bugs exposed by this example:

- [ ] If there is a new line before the docstring, then the boxes precede the code by one line
- [ ] If the comment preceding a line excedes 60 characters (limit of linewrap), the boxes are not positioned correctly horizontally.

**Comment by rlisahuang** [comment]
Creating a separate issue for the PB positioning bug with long comments.

**Comment by rlisahuang** [comment]
Fixed in #29
